README.md

```diff
@@ -445,6 +456,15 @@ And finally execute one of the commands mentioned in the section
   journal={arXiv preprint arXiv:2106.TODO},
   year={2021}
 }
+
+@misc{chen2021vision,
```

Comment:
Maybe `chen2021sam` or `chen2021outperform`?
(Not sure what's customary, but I find it useful to have the most salient word in the reference)